### ISSUE TITLE
Random Mode

### ISSUE THREAD

**Issue by lighterletter** [RosmaryFC/Meme-ify-Me-ARHM#43]
Would be cool to implement a random feature that lets you add random words of text to your chosen picture. Or if you want to go the full monty, use popular templates and randomize everything. That was my original plan but I did not have time to try and implement it.

**Comment by RosmaryFC** [comment]
such as the doge meme?

**Comment by lighterletter** [comment]
No, I thought a general roll through the well known memes, with arrays of relevant words pertaining to each. Then random cheesy punchlines. But that's also a design decision. Since it's your app there's no pressure to implement it.
